codegen-server-test/python/model/pokemon-common.smithy

```diff
@@ -1 +0,0 @@
-../../../codegen-core/common-test-models/pokemon-common.smithy
```

Comment:
What changed with this symlink? Its showing up as both deleted and created in the GitHub diff.

Reply:
I have no idea :(

Reply:
I reverted the commit that was causing this. It should work now.